source/administration/monitoring/bucket-notifications.rst

```diff
@@ -211,6 +211,122 @@ You can only listen to these events through the `ListenNotification <https://min
 
 .. todo
 
+
+Payload schema
+--------------
+
+All notification payloads use the same overall schema.
+Depending on the type of notification, some fields may be omitted or have null values.
+
+.. code-block:: json
+
+   {
+       "eventVersion": "string",
+       "eventSource": "string",
+       "awsRegion": "string",
+       "eventTime": "string",
+       "eventName": "string",
+       "userIdentity": {
+           "principalId": "string"
+       },
+       "requestParameters": {
+           "key": "value"
+       },
+       "responseElements": {
+           "key": "value"
+       },
+       "s3": {
+           "s3SchemaVersion": "string",
+           "configurationId": "string",
+           "bucket": {
+               "name": "string",
+               "ownerIdentity": {
+                   "principalId": "string"
+               },
+               "arn": "string"
+           },
+           "object": {
+               "key": "string",
+               "size": 10000,
+               "eTag": "string",
+               "contentType": "string",
+               "userMetadata": {
+                   "key": "string"
+               },
+               "versionId": "string",
+               "sequencer": "string"
+           }
+       },
+       "source": {
+           "host": "string",
+           "port": "string",
+           "userAgent": "string"
+       }
+   }
+
+
+Example
+~~~~~~~
+
+The following example is a notification for an ``s3:ObjectCreated:Put`` event:
+
+.. code-block:: json
+
+   {
+     "EventName": "s3:ObjectCreated:Put",
+     "Key": "test-bucket/image.jpg",
+     "Records": [
+       {
+         "eventVersion": "2.0",
+         "eventSource": "minio:s3",
+         "awsRegion": "",
+         "eventTime": "2025-02-06T01:04:31.998Z",
+         "eventName": "s3:ObjectCreated:Put",
+         "userIdentity": {
+           "principalId": "access_key"
+         },
+         "requestParameters": {
+           "principalId": "access_key",
+           "region": "",
+           "sourceIPAddress": "192.168.1.10"
+         },
+         "responseElements": {
+           "x-amz-id-2": "dd9025bab4ad464b049177c95eb6ebf374d3b3fd1af9251148b658df7ac2e3e8",
+           "x-amz-request-id": "182178E8B36AC9DF",
+           "x-minio-deployment-id": "2369dcb4-348b-4d30-8fc9-61ab089ba4bc",
+           "x-minio-origin-endpoint": "https://minio.test.svc.cluster.local"
```

Comment:
Not sure what a "real" value would look like, as opposed to notifications coming from my toy VM. Is this typically the source host's FQDN?

Reply:
I've used https://webhook.site/ in the past for testing, if you want to see what winds up there from something like play.

Reply:
That's how I got this payload example. Wasn't sure if there was something about my VM setup that would be inaccurate for a real deployment. But then I did it with aistor.min.dev and it looked similar. So now I think this is fine, even with `.local`